pkgs/development/python-modules/inform/default.nix

```
@@ -0,0 +1,36 @@
+{ lib, buildPythonPackage, fetchFromGitHub
+, arrow
+, six
+, hypothesis
+, pytest
+, pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "inform";
+  version = "1.23";
+
+  src = fetchFromGitHub {
+    owner = "KenKundert";
+    repo = "inform";
+    rev = "v${version}";
+    sha256 = "02zlprvidkz51aypss4knhv7dbr0sbpz3caqjzf9am2n1jx2viyy";
+  };
+
+  propagatedBuildInputs = [ arrow six ];
+
+  checkInputs = [ pytest hypothesis ];
+  checkPhase = "./test.doctests.py && ./test.inform.py && pytest";
+
+  meta = with lib; {
+    description = "Print & logging utilities";
```

```suggestion
    description = "Print and logging utilities";
```

No need for fancy characters.